src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -754,4 +754,36 @@ private void shiftQuestionNumbersDown(int questionNumberToShiftFrom,
             }
         }
     }
+    
+    /*
+     * Removes questions with no recipients. Also may modify value for
+     * question.numberOfEntitiesToGiveFeedbackTo
+     */
+    public List<FeedbackQuestionAttributes> getQuestionsWithRecipients(
+            List<FeedbackQuestionAttributes> questions, String giver)
+            throws EntityDoesNotExistException {
+        List<FeedbackQuestionAttributes> questionsWithRecipients = new ArrayList<FeedbackQuestionAttributes>();
+        for (FeedbackQuestionAttributes question : questions) {
+            if (question.numberOfEntitiesToGiveFeedbackTo == Const.MAX_POSSIBLE_RECIPIENTS) {
+                question.numberOfEntitiesToGiveFeedbackTo = this
```

Comment:
@kanghj Yeah, I just did that to save on db operations. Anyways, I will submit an updated PR.